api/coda.py

```diff
@@ -104,20 +104,19 @@ def get_user_row(self, field: str, value: str) -> Optional[dict]:
             column_name=field, value=value
         )
         if rows:
-            return rows[0].to_dict()
+            return rows[0]
 
     def update_user_stamps(self, user: DiscordUser, stamp_count: float) -> None:
-        """Update stamps count in Coda [users/team table](https://coda.io/d/AI-Safety-Info_dfau7sl2hmG/Team_sur3i#_lu_Rc)"""
+        """Update stamps count in Coda 
+        [users/team table](https://coda.io/d/AI-Safety-Info_dfau7sl2hmG/Team_sur3i#_lu_Rc)"""
         # get row
-        rows = self.users.find_row_by_column_name_and_value(
-            column_name="Discord handle", value=get_user_handle(user)
-        )
-        if rows is None:
-            self.log.error(
+        row = self.get_user_row("Discord handle", get_user_handle(user))
+        if row is None:
+            self.log.info(
```

Comment:
TBH, I'd be fine if it was still logged as `error`, just without the exception .. does it happen too often that we want to keep it at `info` level?